packages/mobile/src/components/drawers/RemoveDownloadedCollectionDrawer.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react'
+
+import { useDrawer } from 'app/hooks/useDrawer'
+import { removeCollectionDownload } from 'app/services/offline-downloader'
+
+import { ConfirmationDrawer } from './ConfirmationDrawer'
+
+const messages = {
+  header: 'Are You Sure?',
+  description:
+    'Are you sure you want to remove this Playlist from your device?',
+  confirm: 'Remove Downloaded Playlist'
+}
+
+const drawerName = 'RemoveDownloadedCollection'
+
+export const RemoveDownloadedCollectionDrawer = () => {
+  const { data } = useDrawer(drawerName)
+  const { collectionId, tracksForDownload } = data
+
+  const handleConfirm = useCallback(() => {
+    removeCollectionDownload(collectionId, tracksForDownload)
+  }, [collectionId, tracksForDownload])
+
+  return (
+    <ConfirmationDrawer
+      drawerName={drawerName}
+      messages={messages}
+      onConfirm={handleConfirm}
```

Comment:
if the user clicks off the modal, that'll trigger `onCancel` and do nothing, is that correct?

Reply:
yeah exactly, the NativeDrawer simply calls onClose